datasource.go

```diff
@@ -24,14 +24,17 @@ package sdk
 type Datasource struct {
 	ID                uint        `json:"id"`
 	OrgID             uint        `json:"orgId"`
+	Uid               string      `json:"uid"`
```

Comment:
Let's call it `UID` for consistency (https://github.com/grafana-tools/sdk/blob/f7c763c3738c279f7e926c5a3bc8353b4080312b/folder.go#L26)

Reply:
Fixed